src/app/utils.py

```diff
@@ -518,26 +518,22 @@ def check_spdx_license(licenseText):
     spdxLicenseTexts = r.mget(spdxLicenseIds)
     licenseData = dict(list(zip(spdxLicenseIds, spdxLicenseTexts)))
     matches = get_close_matches(licenseText, licenseData)
-
     if not matches:
         matchedLicenseIds = None
         matchType = 'No match'
-    
-    elif 1.0 in list(matches.values()) or all(0.99 < score for score in list(matches.values())):
-        matchedLicenseIds = list(matches.keys())
-        matchType = 'Perfect match'
```

Comment:
This seems strange. If there is one match with a `1.0` we include all matches in the result (also bad matched). And if all matches are over `0.99` we return them all. In general, sometimes the function returns a list and sometimes single values.

Reply:
Yes, agreed. This is strange and incorrect.